src/server/config.js

```diff
@@ -5,23 +5,36 @@ import cjson from 'cjson';
 // avoid ESLint errors
 const logger = console;
 
+// Tries to load a .babelrc and returns the parsed object if successful
+function loadBabelConfig(babelConfigPath) {
+  let config;
+  if (fs.existsSync(babelConfigPath)) {
+    const content = fs.readFileSync(babelConfigPath, 'utf-8');
+    try {
+      config = cjson.parse(content);
+      config.babelrc = false;
```

Comment:
If you don't specifically set this option (babelrc=false), the babel-loader
will look for a .babelrc at the module root and try to use it anyways,
ignoring options you pass to the loader.

On Tue, Apr 26, 2016 at 2:48 AM, Arunoda Susiripala <
notifications@github.com> wrote:

> In src/server/config.js
> <https://github.com/kadirahq/react-storybook/pull/149#discussion_r61057793>
> :
>
> > @@ -5,23 +5,36 @@ import cjson from 'cjson';
> >  // avoid ESLint errors
> >  const logger = console;
> >
> > +// Tries to load a .babelrc and returns the parsed object if successful
> > +function loadBabelConfig(babelConfigPath) {
> > +  let config;
> > +  if (fs.existsSync(babelConfigPath)) {
> > +    const content = fs.readFileSync(babelConfigPath, 'utf-8');
> > +    try {
> > +      config = cjson.parse(content);
> > +      config.babelrc = false;
>
> What's the meaning of this?
> Can't we use this for everything since we load and pass the babelrc
> manually anyway.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kadirahq/react-storybook/pull/149/files/c1d198a96de6c64b3443b5b51e9fa50234cbe4bd#r61057793>
>



-- 
Seth Kinast
http://sethkinast.com/